docs/docsite/rst/reference_appendices/test_strategies.rst

```diff
@@ -46,7 +46,7 @@ In the above setup, `--check` mode in Ansible can be used as a layer of testing
 existing system, using the `--check` flag to the `ansible` command will report if Ansible thinks it would have had to have made any changes to
 bring the system into a desired state.
 
-This can let you know up front if there is any need to deploy onto the given system.  Ordinarily scripts and commands don't run in check mode, so if you
+This can let you know upfront if there is any need to deploy onto the given system.  Ordinarily, scripts and commands don't run in check mode, so if you
```

Comment:
```suggestion
This can let you know up front if there is any need to deploy onto the given system.  Ordinarily, scripts and commands don't run in check mode, so if you
```